api/routes/roles/index.test.js

```diff
@@ -0,0 +1,26 @@
+const tap = require('tap');
+const sinon = require('sinon');
+
+const rolesIndex = require('./index');
+
+tap.test('roles endpoint setup', async endpointTest => {
+  const app = {};
+  const getEndpoint = sinon.spy();
+  const postEndpoint = sinon.spy();
+  const putEndpoint = sinon.spy();
+
+  rolesIndex(app, getEndpoint, postEndpoint, putEndpoint);
+
+  endpointTest.ok(
+    getEndpoint.calledWith(app),
+    'users GET endpoint is setup with the app'
+  );
+  // endpointTest.ok(
```

Comment:
what's the thinking here - to uncomment these once the post and put routes are filled in?

Reply:
Yeah, I originally starting writing the code for all the endpoints in here, but then I realized I should probably do them one at a time.  So...  it's not as clean as I'd like, but it will be soon.  😛